src/Storage/Storage.Management/help/Start-AzStorageBlobCopy.md

```diff
@@ -16,7 +16,8 @@ Starts to copy a blob.
 ### ContainerName (Default)
 ```
 Start-AzStorageBlobCopy [-SrcBlob] <String> -SrcContainer <String> -DestContainer <String> [-DestBlob <String>]
- [-PremiumPageBlobTier <PremiumPageBlobTier>] [-Context <IStorageContext>] [-DestContext <IStorageContext>]
+ [-PremiumPageBlobTier <PremiumPageBlobTier>] [-StandardBlobTier <StandardBlobTier>]
```

Comment:
Shouldn't PremiumPageBlobTier and StandardBlobTier be in separate parameter sets?

Reply:
There's already 10 parameter sets of this cmdlet. I don't want to double the parameter sets number, this will make the code difficult to maintain.
I have added code to check the blobtier is aligned with blob type (StandardBlobTier  is only for BlockBlob), so customer will get meaningful/actionable error when use both of them by mistake. (See below sample)

How do you think about it?

```powershell
PS C:\temp> Start-AzStorageBlobCopy -srcContainer $containerName -SrcBlob test.txt -DestContainer $containerName -DestBlob testtier.txt -Context $ctx -DestContext $ctx -Force -StandardBlobTier hot -PremiumPageBlobTier P20
Start-AzStorageBlobCopy : PremiumPageBlobTier can only be set to Page Blob. The Current BlobType is: BlockBlob
Parameter name: BlobType, PageBlobTier
At line:1 char:1
+ Start-AzStorageBlobCopy -srcContainer $containerName -SrcBlob test.tx ...
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidArgument: (:) [Start-AzStorageBlobCopy], ArgumentOutOfRangeException
    + FullyQualifiedErrorId : ArgumentOutOfRangeException,Microsoft.WindowsAzure.Commands.Storage.Blob.Cmdlet.StartAzureStorageBlobCopy
```